src/mocks/Data/Flights.ts

```diff
@@ -1,3 +1,39 @@
 export const FlightData = [
```

Comment:
in `Comments.ts` you're using PascalCase, in `FlightNames.ts` you're using camelCase`, here you're using Pascal again? why?

Reply:
I will correct it.